app/assets/fonts/Canterbury/Canterbury.svg

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Like how we uploaded images to AWS S3 and linked them to our app, we can do the same for fonts. So the fonts files can be accessed both locally and on production.